### PR TITLE
sdk/js: version bump

### PR DESCRIPTION
**Pull request by evan-gray**

### CODE REVIEW

**Review by heyitaki — DISMISSED** [review]

**Review by kev1n-peters — APPROVED** [review]

File: sdk/js/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 0.9.15
```
Comment (tonyjin) [inline sdk/js/CHANGELOG.md:3]: 0.9.16 :)

File: sdk/js/src/token_bridge/redeem.ts [hunk 2]
```diff
@@ -42,17 +42,17 @@ import {
...
   CHAIN_ID_SOLANA,
-  hashLookup,
+  ChainId,
   MAX_VAA_DECIMALS,
+  callFunctionNear,
+  hashLookup,
   uint8ArrayToHex,
 } from "../utils";
-import { parseTokenTransferVaa, SignedVaa } from "../vaa";
+import { SignedVaa, parseTokenTransferVaa } from "../vaa";
```
Comment (tonyjin) [inline sdk/js/src/token_bridge/redeem.ts:55]: ❓

File: sdk/js/src/token_bridge/redeem.ts [hunk 3]
```diff
@@ -27,7 +27,7 @@ import { MsgExecuteContract as XplaMsgExecuteContract } from "@xpla/xpla.js";
 import { Algodv2 } from "algosdk";
 import { AptosClient, Types } from "aptos";
 import BN from "bn.js";
-import { ethers, Overrides } from "ethers";
+import { Overrides, ethers } from "ethers";
```
Comment (tonyjin) [inline sdk/js/src/token_bridge/redeem.ts:30]: Real opinionated, eh?

**Review by heyitaki — APPROVED** [review]

File: sdk/js/CHANGELOG.md (hunk 1 above)
Comment (evan-gray) [reply sdk/js/CHANGELOG.md]: thanks!

File: sdk/js/src/token_bridge/redeem.ts (hunk 2 above)
Comment (evan-gray) [reply sdk/js/src/token_bridge/redeem.ts:55]: lol option+shift+o

File: sdk/js/src/token_bridge/redeem.ts (hunk 3 above)
Comment (evan-gray) [reply sdk/js/src/token_bridge/redeem.ts:30]: same

**Review by heyitaki — APPROVED** [review]

**Review by kev1n-peters — APPROVED** [review]

**Pull request merged**